cypress/integration/dashboard/dashboard_spec.js

```diff
@@ -92,54 +130,162 @@ describe('Dashboard', () => {
     });
   });
 
-
   describe('Textbox', () => {
-    before(function () {
-      cy.login();
-      createNewDashboardByAPI('Foo Bar')
-        .then(slug => `/dashboard/${slug}`)
-        .as('dashboardUrl');
+    beforeEach(function () {
```

Comment:
I think you have some changes in this PR not related to it like this one, right?

Reply:
Should rebase.